.circleci/config.yml

```diff
@@ -342,9 +342,11 @@ steps-mksnapshot-build: &steps-mksnapshot-build
         command: |
           cd src
           ninja -C out/native_mksnapshot v8:mksnapshot
+          ninja -C out/native_mksnapshot electron:electron_mksnapshot_zip
+          cp out/mksnapshot/mksnapshot.zip out/native_mksnapshot/mksnapshot_native.zip
     - store_artifacts:
-        path: src/out/native_mksnapshot/mksnapshot
-        destination: mksnapshot
+        path: src/out/native_mksnapshot/native_mksnapshot.zip
+        destination: native_mksnapshot
```

Comment:
Maybe it would be better to keep the file extension? A zip file without an extension looks odd.

Reply:
keep the extension? in the diff above i added one, do you mean it should be removed?

Reply:
I mean that you create a zip file `native_mksnapshot.zip` and save it as an artifact with name `native_mksnapshot` . A zip file without an extension looks odd.